Client/src/Components/Graph/Graph.js

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react'
+import * as d3 from "d3"
+
+//Global variables for graph creation
+
+var margin = {
+    top: 50,
+    right: 210,
+    bottom: 50,
+    left: 70
+},
```

Comment:
I think we should be using consts instead of vars

Reply:
The margins, outers and inners are changed to const, but the xscale and axes will change will scaling so it is left as a var